src/marine/cryosat_ice2ioda.py

```diff
@@ -18,7 +18,7 @@
 import os
 from pathlib import Path
 
-IODA_CONV_PATH = Path(__file__).parent/"@SCRIPT_LIB_PATH@"
+IODA_CONV_PATH = Path(__file__).parent/"../lib/pyiodaconv"
```

Comment:
This code shouldn't change.  The CMake build and install processes will be substituting the `@SCRIPT_LIB_PATH@` variable.

Reply:
Thank you for letting me know about it. I brought SCRIPT_LIB_PATH back.

Reply:
@mjkagnes123 thank you for addressing this. @markjolah I'm going to reset your review - please review again. Thanks!